lib/pure/uri.nim

```diff
@@ -496,4 +581,50 @@ when isMainModule:
     doAssert "https://example.com/about".parseUri().isAbsolute == true
     doAssert "https://example.com/about/staff.html".parseUri().isAbsolute == true
     doAssert "https://example.com/about/staff.html?".parseUri().isAbsolute == true
-    doAssert "https://example.com/about/staff.html?parameters".parseUri().isAbsolute == true
\ No newline at end of file
+    doAssert "https://example.com/about/staff.html?parameters".parseUri().isAbsolute == true
+
+  block: # parseQuery & newQuery tests
+    assert "foo=baa&baz=bahhz&baz=bahhz2".parseQuery() == 
+        { "foo": "baa", "baz": "bahhz", "baz": "bahhz2"}.newOrderedTable()
+    assert "foo=baa&baz=bahhz&baz=bahhz2".parseQuery() == 
+      {"foo": "baa", "baz": "bahhz", "baz": "bahhz2"}.newOrderedTable()
```

Comment:
OrderedTable can handle multiple keys (i think this was/is not very well known cause tables was a little buggy, i've fixed what i found in my previous pr)